microsoft-edge/extensions-chromium/publish/create-dev-account.md

```diff
@@ -1,39 +1,88 @@
+
+1. Select **Create a Microsoft account**.
+
 1. Complete the registration steps.
-    
-If you have a developer account registered on Partner Center, use the corresponding Microsoft account \(MSA\) to sign into your developer account. Then enroll in the Microsoft Edge program. 
+
+If you have a Partner Center account for which the Primary Owner is a Microsoft account (MSA), use that a Microsoft account (MSA) to sign into your Partner Center account.  Then enroll in the Microsoft Edge program.
 
 > [!NOTE]
-> The Microsoft Edge extensions does not currently support registering with a work or school account. You need to register using an MSA account and then link your Azure AD tenants with that account for extension management. 
+> The Microsoft Edge program does not currently support registering with a work or school account.  You must register using a Microsoft account (MSA) and then link your Azure AD tenants with that account, to be able to manage extensions.
 
-## Enroll in the Microsoft Edge program on Partner Center  
 
-1.  Navigate to the [developer page][MicrosoftPartnerCenter] and choose **Partner Center**.  
-1.  If you have a Microsoft account, sign into it now.  If not, create a new Microsoft account. Use the same Microsoft account that you use to sign into your developer account.  After you sign in, a registration form is displayed. The following table describes the fields on the registration form.  
-    
-    To enroll in the Microsoft Edge program, sign in to your account and complete the form.  
+<!-- ====================================================================== -->
+## Enroll in the Microsoft Edge program on Partner Center
+
+<!-- 1.  Navigate to the [webpage about Partner Center](https://partner.microsoft.com).  You might see a "Join the Microsoft Partner Network" page with a **Become a partner** button, or a "Welcome back" page with a **Visit Partner Center** button.  Select the **Become a partner** button or the **Visit Partner Center** button. -->
+
+<!-- todo: test this step -->
+1.  Navigate to the [Partner Center developer login page](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), and then select **Partner Center**.
+
+1.  If you have a Microsoft account (MSA), sign into it now.  Use the same Microsoft account (MSA) that you use to sign into your Partner Center account.  A _Microsoft account (MSA)_ is an Outlook.com, Live.com, or Hotmail.com account.
+
+1.  If you don't have a Microsoft account (MSA), either create a new Microsoft account (MSA) directly, or login to Partner Center by using your GitHub account using the next step.  The Partner Center account must have a Primary Owner that's a Microsoft account (MSA).  After you sign in, a registration form is displayed.  Use the following table to help you fill in the fields in the registration form.
```

Comment:
Currently it seems like the login with Github and filling the form step is mixed up. Should we break down point 3 and 4 two parts as suggested below?

3. If you don't have a Microsoft account (MSA), either create a new Microsoft account (MSA) directly, or login to Partner Center by using your GitHub account using the next step. If you want to login to Partner Center by using your existing personal GitHub account, open the article Publish Microsoft Edge extensions by using a GitHub account in a new tab or window, and follow the steps. Your GitHub account will be linked to an automatically created Microsoft account (MSA) whose credentials you can use to register for the Microsoft Edge program. 

4. After you sign in, a registration form is displayed. Use the following table to help you fill in the fields in the registration form.